controllers/IssueController.js

```diff
@@ -130,18 +171,21 @@ const addIssue = async (req, res) => {
         title,
         description,
         section,
+        images,
         department,
         scope,
         id
       );
+      console.log("issue created");
+      await updateImageIssueId(images, id);
```

Comment:
`images` can be null if no image is uploaded for an issue and it might give errors.

Reply:
Nope. $in would not work if there are no matches. Hence I made images as required.

Reply:
If there are no images we shouldn't call this method in the first place.

Reply:
Okay. Can be checked at the time of issue creation.